source/Preferences.cpp

```diff
@@ -37,6 +37,9 @@ namespace {
 	int zoomIndex = 4;
 	constexpr double VOLUME_SCALE = .25;
 
+	int screenModeIndex = 0;
+	const vector<string> SCREEN_MODE_SETTINGS = {"windowed", "fullscreen"};
```

Comment:
```suggestion
	const vector<string> SCREEN_MODE_SETTINGS = { "windowed", "fullscreen" };
```

Optional style suggestion.

Reply:
I don't think we add spaces between braces when initializing.